community/chef-infra-server.mql.yaml

```diff
@@ -1,44 +1,38 @@
 policies:
   - uid: chef-infra-server
     name: Chef Infra Server Policy
-    version: "1.0.0"
-    authors:
-      - name: Tim Smith
-        email: tim@mondoo.com
+    version: 1.0.0
+    license: unspecified
     tags:
       mondoo.com/category: security
       mondoo.com/platform: linux
+    authors:
+      - name: Tim Smith
+        email: tim@mondoo.com
     docs:
-      desc: |-
-        Chef Infra Server Policy identifies several misconfigurations and end of life components that allow attackers to expose node information:
-          - Insecure disk permissions on critical directories and configuration files.
-          - End of life components installed on the Chef Infra Server such as Push Jobs, Analytics, or Reporting, which no longer receive security updates.
-          - Insecure servers settings such non-secure TLS support or legacy add-on compatibility.
-          
-        If you have questions, comments, or have identified ways to improve this policy, please write me at tim@mondoo.com, or reach out in the [Mondoo Slack Community](https://mondoo.link/slack).
-    specs:
-      - asset_filter:
-          query: |
-            platform.family.contains(_ == 'linux')
-            file("/opt/opscode/").exists
-        scoring_queries:
-          etc-opscode-directory-permissions:
-          chef-server-rb-permissions:
-          non-eol-infra-server:
-          eol-reporting-addon:
-          eol-push-jobs-addon:
-          eol-analytics-addon:
-          eol-ha-addon:
-          pivotal-pem-permissions:
-          webui-pem-permissions:
-          secure-tls-only:
-          secrets-file-permissions:
-          disable-insecure-addon-compat:
+      desc: "Chef Infra Server Policy identifies several misconfigurations and end of life components that allow attackers to expose node information:\n  - Insecure disk permissions on critical directories and configuration files.\n  - End of life components installed on the Chef Infra Server such as Push Jobs, Analytics, or Reporting, which no longer receive security updates.\n  - Insecure servers settings such non-secure TLS support or legacy add-on compatibility.\n  \nIf you have questions, comments, or have identified ways to improve this policy, please write me at tim@mondoo.com, or reach out in the [Mondoo Slack Community](https://mondoo.link/slack)."
```

Comment:
@chris-rock Any way we could get the automatic formatting to use `|-` when there are line breaks in the text? These become super hard to hand edit now.

Reply:
Yeah, unclear why this happens because it does not happen all the time.